jsk_footstep_planner/euslisp/hoge.l

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env roseus
```

Comment:
please change file name

Reply:
I have removed this file because it is redundant file.